.github/workflows/cicd.yaml

```diff
@@ -113,58 +128,66 @@ jobs:
     runs-on: ubuntu-latest
     container:
       image: docker:stable
-    needs: [pytest]
+    needs: [build]
     steps:
       - uses: actions/checkout@v2
+      - uses: actions/download-artifact@v2
+        with:
+          name: images
       - name: Install trivy
         run: |
           apk update
           apk add curl
           curl -sfL https://raw.githubusercontent.com/aquasecurity/trivy/master/contrib/install.sh | sh -s -- -b /usr/local/bin
-      - name: Build image
-        run: docker build -t $GITHUB_SHA -f docker/Dockerfile .
-      - name: Save image
-        run: docker save $GITHUB_SHA -o $GITHUB_SHA.tar
-      - name: Scan
-        run: trivy image --input $GITHUB_SHA.tar -o trivy-report.json --exit-code 1 --severity="UNKNOWN,MEDIUM,HIGH,CRITICAL"
+          mkdir trivy-reports
+      - name: Scan Image
+        run: trivy image --input ${GITHUB_SHA}_image.tar
+                -o trivy-reports/image.txt
+                --exit-code 1
+                --severity="UNKNOWN,MEDIUM,HIGH,CRITICAL"
+      - name: Scan Hook
+        run: trivy image --input ${GITHUB_SHA}_hook.tar 
+                -o trivy-reports/hook.txt
+                --exit-code 1
+                --severity="UNKNOWN,MEDIUM,HIGH,CRITICAL"
       - uses: actions/upload-artifact@v2
         if: failure()
         with:
-           name: trivy-report
-           path: trivy-report.json
+           name: trivy-reports
+           path: trivy-reports
 
   integration-test:
     runs-on: ubuntu-latest
-    needs: [bandit, safety, trivy]
+    needs: [build]
     steps:
       - uses: actions/checkout@v2
-
+      - uses: actions/download-artifact@v2
+        with:
+          name: images
+      - name: Load Docker images
+        run: |
+          docker load -i ${GITHUB_SHA}_image.tar
+          docker load -i ${GITHUB_SHA}_hook.tar
       - name: Install yq and bash
         run: |
           sudo snap install yq
           sudo apt update
           sudo apt install bash -y
-
-      - name: Build Connaisseur image
+      - name: Set version
```

Comment:
Since the vars set in this are only used during `make docker` this section can be dropped completely